framework/source/class/qx/ui/table/model/Simple.js

```diff
@@ -71,6 +71,8 @@ qx.Class.define("qx.ui.table.model.Simple",
           return result;
         }
       }
+      if(obj1==null && obj2!==null) return -1;
+      if(obj2==null && obj1!==null) return 1;
```

Comment:
These will match any falsy value of obj1 and obj2, e.g., `false` or `0`,  with `==`. The part before `&&` on lines 74 and 75 must use `===` instead of `==`.

Reply:
Similarly, in all cases below.

Reply:
> These will match any falsy value of obj1 and obj2, e.g., `false` or `0`, with `==`. The part before `&&` on lines 74 and 75 must use `===` instead of `==`.

No, thats wrong.

obj1==null will only return true if obj1 is null or undefined, which is exactly the check I intended to use.

Reply:
```js
console.log(null==0) // false
console.log(null==false) // false
console.log(null==null) // true
console.log(null==undefined) // true
```

Reply:
@derrell is this really a breaking change?  it sounds like the table sort works fine except when nulls are involved, in which case the sort order is random(ish) and undesirable in all cases

Reply:
@adrelino I stand corrected. Using `===` still seems prudent with falsy things, but I see that my statement was incorrect.

@johnspackman It's breaking only in that null was treated accidentally previously that is now explicit and possibly different.

I don't think it need be considered a breaking change.